config-model/src/main/java/com/yahoo/vespa/model/admin/clustercontroller/ClusterControllerContainer.java

```diff
@@ -110,7 +110,10 @@ public void getConfig(ZookeeperServerConfig.Builder builder) {
 
     @Override
     public void getConfig(QrStartConfig.Builder builder) {
-        builder.jvm(new QrStartConfig.Jvm.Builder().heapsize(512));
+        builder.jvm.heapsize(512);
+        builder.jvm.heapSizeAsPercentageOfPhysicalMemory(0);
+        builder.jvm.availableProcessors(2);
```

Comment:
Note that the builder setters return the builder itself, so it's not necessary to repeate `builder.` for every setting.

Reply:
Fixed